src/reflect/scala/reflect/internal/Symbols.scala

```diff
@@ -2098,18 +2098,26 @@ trait Symbols extends api.Symbols { self: SymbolTable =>
       // handling of non-public parameters seems to change the order (see scala/bug#7035.)
       //
       // Luckily, the constrParamAccessors are still sorted properly, so sort the field-accessors using them
-      // (need to undo name-mangling, including the sneaky trailing whitespace)
+      // (need to undo name-mangling, including the sneaky trailing whitespace, and match longest first)
       //
       // The slightly more principled approach of using the paramss of the
```

Comment:
small old typo --> The slightly more principled approach of using the **params** of the

Reply:
Not 100% sure, but it's probably not a type. `paramss` is used in the compiler to refer to multiple parameter lists, so `List[List[Param]]`.